literature.bib

```diff
@@ -1241,6 +1276,67 @@ @proceedings{VaMoS23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2022 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{SC:AS22,
+  author  = {Sepúlveda, Samuel and Cravero, Ania},
+  title   = {{Reasoning Algorithms on Feature Modeling -- A Systematic Mapping Study}},
+  journal = {Applied Sciences},
+  volume  = {12},
+  year    = 2022
+}
+
+@article{LRI+:WIREs22,
+  author  = {Le Quy, Tai and Roy, Arjun and Iosifidis, Vasileios and Zhang, Wenbin and Ntoutsi, Eirini},
+  title   = {{A Survey On Datasets For Fairness-Aware Machine Learning}},
+  journal = {WIREs Data Mining and Knowledge Discovery},
+  volume  = {12},
+  number  = {3},
+  doi     = {https://doi.org/10.1002/widm.1452},
+  year    = 2022
+}
+
+@inproceedings{HGP+:MODEVAR22,
```

Comment:
we already have this one as HGP+:SPLC22. probably both entries should be merged, and MODEVAR is probably more accurate than SPLC